### PR TITLE
fix: move onnxruntime import to function scope

### PR DESCRIPTION
**Pull request by justinchuby**
Move ort import into functions so onnx script still works without ort. This is also more friendly for new potential evaluators that do not require ORT.

fixes: #81

### CODE REVIEW

File: onnxscript/evaluator.py [hunk 1]
```diff
@@ -7,19 +7,17 @@
...
-from onnxruntime.capi.onnxruntime_pybind11_state import (
-    Fail,
-    InvalidArgument,
-    InvalidGraph,
-)
 
 from onnxscript import autocast, irbuilder, onnx_opset, tensor, utils, values
 
+if typing.TYPE_CHECKING:
+    import onnxruntime as ort
```
Comment (gramalingam) [inline onnxscript/evaluator.py:19]: Just wondering: why is this necessary?

File: onnxscript/values.py [hunk 2]
```diff
@@ -13,7 +13,7 @@
 import numpy as np
 import onnx
 
-from onnxscript import debuginfo, irbuilder, tensor
+from onnxscript import debuginfo, evaluator, irbuilder, tensor
```
[the remark this answers is not in the document]
Comment (gramalingam) [reply onnxscript/values.py:16]: Why not import evaluator down below, when needed? It avoids the import cycle warning above, and tells us something about where exactly it is used, which will be useful if we want to do any refactoring to eliminate more of these cycle warnings.

[the remark this answers is not in the document]
Comment (gramalingam) [reply onnxscript/values.py:16]: Oh, I see, it is for the type annotation: I guess you could enclose `evaluator.Evaluator` in a string? But, anyway, I think importing at top is fine too, except for these cycle warnings

**Review by gramalingam — APPROVED** [review]

File: onnxscript/values.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (justinchuby) [reply onnxscript/values.py]: I did it follow the general recommendation of importing at the top. having imports in between code generally makes the code harder to manage as we move things around. I will see if there is a way to handle the cycles

File: onnxscript/evaluator.py (hunk 1 above)
Comment (justinchuby) [reply onnxscript/evaluator.py:19]: This is for type annotation (for `_cache_models`). ort will be imported by type checkers but not in run time, so we don't fail if the environment doesn't have ort.

File: onnxscript/values.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (justinchuby) [reply onnxscript/values.py]: ok this doesn't work yet because the generated code is importing objects directly.

`from onnxscript.values import Op, Opset` etc.

we may need to clean that up first
